packages/devextreme-react/src/core/__tests__/template.test.tsx

```diff
@@ -1,11 +1,18 @@
 /* eslint-disable max-classes-per-file */
 import * as events from 'devextreme/events';
 import { Component } from '../component';
-
 import ConfigurationComponent from '../nested-option';
 import { Template } from '../template';
-import { mount, React, shallow } from './setup';
-import { TestComponent, Widget, WidgetClass } from './test-component';
+import { cleanup, render, screen } from '@testing-library/react';
+import * as React from 'react';
+import {
+  TestComponent,
+  Widget,
+  WidgetClass
+} from './test-component';
+import { createRef } from 'react';
```

Comment:
You have already imported react
https://github.com/DevExpress/devextreme-react/pull/529/files#diff-1b74b0079e640e3c313b9f5123f4aaaeaa4f27d3b38c80a08aa666014ab2f775R7

Reply:
I deleted superfluous